lib/mayaUsd/ufe/Global.cpp

```diff
@@ -115,11 +118,17 @@ MStatus initialize()
 	UFE_V2(auto usdAttributesHandler = UsdAttributesHandler::create();)
 	UFE_V2(auto usdObject3dHandler = UsdObject3dHandler::create();)
 	UFE_V2(auto usdContextOpsHandler = UsdContextOpsHandler::create();)
+#if UFE_PREVIEW_VERSION_NUM >= 2022
+	UFE_V2(auto usdUIInfoHandler = UsdUIInfoHandler::create();)
+#endif
```

Comment:
A lot of my code will be double protected: UFE_V2 and UFE preview version 0.2.22